new-components/scanners/bandit/.env

```diff
@@ -0,0 +1,6 @@
+# This is for local setup only.
+SMITHY_INSTANCE_ID=8d719c1c-c569-4078-87b3-4951bd4012ee
+SMITHY_LOG_LEVEL=debug
+SMITHY_IS_LOCAL=false
+SMITHY_STORE_TYPE=postgresql
```

Comment:
You can remove both of this and the following env var so that the component will use SQLite out of the box.